Delta/Emulation/GameViewController.swift

```diff
@@ -281,6 +281,7 @@ extension GameViewController
         self.view.layoutIfNeeded()
         
         self.controllerView.translucentControllerSkinOpacity = Settings.translucentControllerSkinOpacity
+        self.controllerView.isControllerSkinDebugModeEnabled = Settings.isControllerSkinDebugModeEnabled
```

Comment:
As mentioned in DeltaCore PR, I think this should be `.isDebugModeEnabled`
```suggestion
        self.controllerView.isDebugModeEnabled = Settings.isControllerSkinDebugModeEnabled
```